inc/TrkInfo.hh

```diff
@@ -86,11 +86,6 @@ namespace mu2e
     int  ndigigood =0; // number of true straw digitizations where the primary particle (not a delta-ray) crossed threshold
     int nhits =0, nactive =0; // number of hits on the track and active on the track generated by the primary particle
     int nambig =0; // number of true hits where the reconstruction assigned the correct left-right ambiguity
-    int pdg =0, gen =0, proc =0; // true PDG code, generator code, and process code of the primary particle
-    float otime =0;  // origin time
-    MCRelationship prel =MCRelationship::none; // relationship if this tracks primary particle to the event primary
```

Comment:
the primary MCRelationship value needs to be stored somewhere, probably in siminfo now.  This would allow eliminating the demmcpri branch, as discussed in the comments.